core/app/services/spree/cart/destroy.rb

```diff
@@ -0,0 +1,36 @@
+module Spree
+  module Cart
+    class Destroy
```

Comment:
@KacperMekarski `Destroy` service should destroy cart, we have already an `order.empty!` method which needs to be moved to it's own `Empty` service

Reply:
@damianlegawiec I have created already service Spree::Cart::Empty which is used in `order.empty!` (you gave me a note above how it should look like.

Regarding to `Spree::Cart::Destroy`, it should only destroy cart, right? In that case I think I should add some validations to check if order does not have line items, adjustments, shipments, state_changes, order_promotions (everything what is destroyed during `order.empty!`.

Reply:
@KacperMekarski 

`Spree::Cart::Empty` -> this needs to hold contents of the current `Order#empty!` method
`Spree::Cart::Destroy` -> this needs to call `Oder#destroy` for now - maybe additional logic in the future

Reply:
@damianlegawiec done